assets/js/components/ParticipantsList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  SimpleGrid,
+  Box,
+  Table,
+  Flex,
+  Heading,
+  ButtonGroup,
+  Button,
+  Spacer,
+  Avatar,
+  Menu,
+  IconButton,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+
+const ParticipantMenu = (props) => {
+  return (
+    <Menu>
+      <MenuButton as={IconButton} aroa-label="Options" icon={"Menu Icon"} variant="outline" />
+      <MenuList>
+        <MenuItem>Opcja 1</MenuItem>
+        <MenuItem>Opcja 2</MenuItem>
+        <MenuItem>Opcja 3</MenuItem>
+        <MenuItem>Opcja 4</MenuItem>
+        <MenuItem>Opcja 5</MenuItem>
+      </MenuList>
+    </Menu>
+  );
+};
+
+const Participant = ({ name }) => {
+  return (
+    <Flex width="15vw" alignItems="center" gap="1">
+      <Avatar name="Dan Abrahmov" src="https://bit.ly/dan-abramov" />
+      <Box p="2">
+        <Heading size="md">{name}</Heading>
+      </Box>
+      <Spacer />
+      <ParticipantMenu />
```

Comment:
I would also think about that moderator and viewer will have different options available (I am not sure if viewer will have any options). So I would add some logic about changing options based on type of the user.

Reply:
Added prop to the component telling if user is a moderator.